scripts/deploy_localnet.py

```diff
@@ -4,6 +4,7 @@
     MockToken,
```

Comment:
No need to commit this file, rebase on master as these changes are already there

Reply:
I had conflict, i was forced to commit